src/devices/LidarLiteV3/samples/README.md

```diff
@@ -0,0 +1,195 @@
+                Console.WriteLine($"Current Distance: {currentDistance} cm");
+                Thread.Sleep(1000);
+            }
+        }
+    }
+    private static I2cDevice CreateI2cDevice()
+    {
+        var settings = new I2cConnectionSettings(1, LidarLiteV3.DefaultI2cAddress);
+        return I2cDevice.Create(settings);
+    }
+}
+```
+
+# Advance Usage
+
+## Power Modes
+
+Power can be controlled to the device via a GPIO pin. Use the optional constructor parameters to 
+specify the GPIO controller and a power enable pin number (numbering scheme depends on GpioController).
+
+```csharp
+static void Main(string[] args)
+{
+    int powerEnablePin = 13;
+    using (var llv3 = new LidarLiteV3(CreateI2cDevice(), new GpioController(), powerEnablePin))
+    {
+        // Power off the device.
+        llv3.PowerOff();
+        // Device is completely turned off.
+        Console.WriteLine("Device is off.");
+        Thread.Sleep(5000);
+        // Power on the device, device is ready in ~22ms.
+        Console.WriteLine("Device is on.");
+        llv3.PowerOn();
+        // Sleep 50ms.
+        Thread.Sleep(50);
+        // Get a reading.
+        ushort currentDistance = llv3.MeasureDistance();
+        Console.WriteLine($"Current Distance: {currentDistance} cm");
+    }
+}
+```
+
+It's also possible to disable the receiver circuit (saving 40 mA) or put the device to sleep 
+(saving 20 mA).  However, it's recommended to use the power enable pin instead
+since the initialization time is only 2 ms shorter.
+
+```csharp
+using (var llv3 = new LidarLiteV3(CreateI2cDevice()))
+{
+   llv3.PowerMode = PowerMode.Sleep;
+}
+```
+
+# Repetition Mode
+
+Instead of getting measurements on-demand, the device can be configure to repeat n number of 
+times, or infinitely.
+
+This is configured via `SetMeasurementRepetitionMode` passing in a mode (`Off`, `Repeat`, or 
+`RepeatInfinitely`), a loop count (if mode is `Repeat`), and a delay between measurements (default to 
+10 hz). A delay of `20` is about `100 hz`.
+
+With a repetition mode set, use `Distance` and `Velocity` to retrieve the current readings in cm.
+Negative velocity is toward the device, positive velocity is away the device.  The unit is 
+cm and depends on the delay. The default 10 hz is about 0.1 m/s.
+
+```csharp
+using (var llv3 = new LidarLiteV3(CreateI2cDevice()))
+{
+    llv3.SetMeasurementRepetitionMode(MeasurementRepetitionMode.RepeatIndefinitely);
+
+    while(true)
+    {
+        Thread.Sleep(5);
+        ushort currentDistanceInCm = llv3.Distance;
+        int currentVelocityInCm = llv3.Velocity;
+    }
+}
+```
+
+# Change the I2C Address
+
+By default, the device has an address of `0x62`.  It's possible to change this address to 
+resolve a conflict with another device or to run multiple devices.
+
+Available addresses are 7-bit values with a 0 in the LSB order.
+
+```csharp
+using (var llv3 = new LidarLiteV3(CreateI2cDevice()))
```

Comment:
So as mentioned before this should be something like:
`new LidarLiteV3(CreateI2cDevice()).SetI2cAddressAndDispose()`